_maps/map_files/Birdshot/birdshot.dmm

```diff
@@ -65799,9 +65799,10 @@
 	pixel_x = -5;
 	pixel_y = 3
 	},
-/obj/machinery/button/door/directional/west{
-	id = "detpriv";
-	name = "Curtains"
+/obj/machinery/button/curtain{
+	pixel_x = -24;
+	name = "Curtains";
+	id = detpriv
```

Comment:
Linters failing because of

![Screenshot (342)](https://github.com/tgstation/tgstation/assets/110812394/611b10df-0a96-4368-ab6e-4fb06950007e)

Wrap it in double quotes "detpriv"